Previous events section

combimauri opened the pull request:
Closes #7 
- Implemented events component.
- Refactored styles.
- Added trailing-comma rule to tslint.
- Added provisional favicon and changed document language to spanish.
- Fixed some little typos.

Additional notes:
- Functionality to schedule event is not implemented yet
- Some styles don't match the design yet, I'm going to fix this in another PR


![Screen Shot 2020-07-28 at 5 50 22 PM](https://user-images.githubusercontent.com/11466673/88725984-f2ac4900-d0fa-11ea-8f0e-aa56773ca6f7.png)
![Screen Shot 2020-07-28 at 5 50 52 PM](https://user-images.githubusercontent.com/11466673/88725992-f5a73980-d0fa-11ea-801a-3a93351d1004.png)

irvin373 commented on src/app/home/banner/banner.component.html:
```diff
@@ -1,34 +1,34 @@
           <div class="row">
             <div class="col s6">
-              <img src="assets/images/wtm.svg" class="right" alt="women tech makers cochabamba"/>
+              <img src="assets/images/wtm.svg" class="right" alt="women tech makers cochabamba">
```
the right wording is `women techmaker cochabamba`

irvin373 commented on src/app/home/invitation/invitation.component.scss:
```diff
@@ -1,33 +1,4 @@
 .text-section {
-  font-size: 20px;
+  font-size: 1.4285rem;
   width: 80%;
```
I don't really sure how browser render text but add `width: 80%` it's ok? In not sure if is apply in a container or in the same text 🙃

combimauri replied:
It should be fine to apply the width directly to the text, but I agree with placing this text and the image besides it inside containers and apply the styles there, I'll work on that

irvin373 replied:
oh ok, in mobile is a recalculate and scaling is not performant 🙈, another type of problems 🙈

combimauri commented on src/index.html:
```diff
@@ -1,13 +1,16 @@
+  <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="icon" type="image/x-icon" href="favicon.ico?v=0.1">
```
WGJ logo is white, so in browsers with light background it's not visible. That's why I used the pixel art girl image. Anyway, favicon should be provided by Carlitos and should be replaced later.

irvin373 replied:
yes, but is important know is temporal :)

combimauri replied:
I could add a TODO tag so it's clear enough. I thought it would be understood because I tagged it as version `0.1` instead of version `1` XD

combimauri commented on src/styles.scss:
```diff
@@ -1,12 +1,10 @@
 
 body {
-  font-family: 'Lato', sans-serif;
+  font-family: 'Open Sans', sans-serif;
```
That's the font used in the design

combimauri commented:
@irvin373 you are right about the commits. I thought about that when I finished the implementation. I'll be more careful about that the next time

combimauri commented:
@irvin373 I fixed the typo in the WTM image and added a TODO for the favicon in my last commit.
About the invitation component and adding width directly to the text, I think this section could be improved further. We could review that later and make the changes in another PR.
